app/src/main/res/values/styles.xml

```diff
@@ -341,17 +341,16 @@
 
     <style name="SimpleDraweeViewPlaceholder">
         <item name="actualImageScaleType">centerCrop</item>
-        <item name="placeholderImage">?attr/thumbnail_image_placeholder</item>
         <item name="placeholderImageScaleType">fitCenter</item>
-        <item name="failureImage">?attr/thumbnail_image_placeholder</item>
+        <item name="failureImage">?android:colorBackground</item>
         <item name="failureImageScaleType">fitCenter</item>
         <item name="fadeDuration">0</item>
-        <item name="backgroundImage">?attr/paper_color</item>
+        <item name="backgroundImage">?android:colorBackground</item>
     </style>
 
     <style name="SimpleDraweeViewPlaceholder.SuggestedEdits">
-        <item name="placeholderImage">@drawable/ic_image_gray_transparent_24dp</item>
```

Comment:
Please also remove the unused resource

Reply:
This will get removed in your PR I guess..

Reply:
Sorry about the misunderstanding. "Delete" unused resource would be a proper word.